client/packages/common/src/ui/forms/JsonForms/useJsonForms.tsx

```diff
@@ -139,6 +169,15 @@ export const useJsonForms = (
     }
   }, [databaseResponse]);
 
+  // user createDoc if there is one
+  useEffect(() => {
+    if (createDoc !== undefined) {
```

Comment:
```suggestion
    if (createDoc) {
```

Is more typical

Reply:
hm works for objects but fails for other types like strings, e.g. `if ("")`, `!== undefined` is clearer on what you want to check

Reply:
similar to above, is empty createdDoc a valid doc ?

Reply:
empty doc evaluates to true, so not a problem here. I wait with these changes till Mark merged his branch, already caused enough conflicts :)